scanners/scanners.go

```diff
@@ -96,8 +96,8 @@ const CustomConfigName = "other-config"
 // CustomConfig ...
 func CustomConfig() (models.BitriseConfigMap, error) {
 	configBuilder := models.NewDefaultConfigBuilder()
-	configBuilder.AppendStepListItemsTo(models.PrimaryWorkflowID, steps.DefaultPrepareStepList(false)...)
-	configBuilder.AppendStepListItemsTo(models.PrimaryWorkflowID, steps.DefaultDeployStepList(false)...)
+	configBuilder.AppendStepListItemsTo(models.PrimaryWorkflowID, steps.DefaultPrepareStepList(steps.PrepareListParams{ShouldIncludeActivateSSH: true})...)
```

Comment:
@ofalvai looks like this is causing an issue for public repositories. We should be checking whether it's a private repo rather than just passing true.

https://bitfall.slack.com/archives/C04D1MH378T/p1678820687391889